gcp/gcp.go

```diff
@@ -13,7 +14,6 @@ import (
 	"github.com/nanovms/ops/types"
 	"golang.org/x/oauth2/google"
 	compute "google.golang.org/api/compute/v1"
-	"google.golang.org/api/dns/v1"
```

Comment:
i'm guessing you have a linter that removed this cause there was no reference to it;

it actually does have a reference though - the pkg name is 'dns' - you can add this line back and prefix it w/ 'dns' like the compute one above it

Reply:
done